From 4e0597e0b3ea3688f341f78d2518aa4ecb1b6e14 Mon Sep 17 00:00:00 2001
From: Khoi Nguyen <khoi.nguyen.aj@renesas.com>
Date: Mon, 2 Aug 2021 13:31:41 +0700
Subject: [PATCH] arm64: dts: renesas: r9a07g054l2-smarc: add drp_reserved and
 v4l2_reserved node

This patch adds memory reserved for DRP-AI and V4L2.

Signed-off-by: Khoi Nguyen <khoi.nguyen.aj@renesas.com>
---
 arch/arm64/boot/dts/renesas/r9a07g054l2-smarc.dts | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r9a07g054l2-smarc.dts b/arch/arm64/boot/dts/renesas/r9a07g054l2-smarc.dts
index 70754e4..94b3e7c6 100644
--- a/arch/arm64/boot/dts/renesas/r9a07g054l2-smarc.dts
+++ b/arch/arm64/boot/dts/renesas/r9a07g054l2-smarc.dts
@@ -36,6 +36,19 @@
 			reusable;
 			reg = <0x00000000 0x68000000 0x0 0x8000000>;
 		};
+
+		/* device specific region for contiguous allocations */
+		drp_reserved: DRP-AI {
+			reusable;
+			reg = <0x0 0x70000000 0x0 0x10000000>;
+		};
+
+		v4l2_reserved: v4l2-DRP_buf@0 {
+			compatible = "shared-dma-pool";
+			reusable;
+			reg = <0x0 0x80000000 0x0 0x00400000>;
+			label = "image_buf0";
+		};
 	};
 
 	mmngr {
-- 
2.7.4

